shell/common/shell_io_manager.cc

```diff
@@ -100,4 +100,11 @@ fml::RefPtr<flutter::SkiaUnrefQueue> ShellIOManager::GetSkiaUnrefQueue() const {
 fml::WeakPtr<ShellIOManager> ShellIOManager::GetWeakPtr() {
   return weak_factory_.GetWeakPtr();
 }
+
+void ShellIOManager::PurgeCaches() const {
+  if (!resource_context_) {
+    return;
```

Comment:
On a debug build shouldn't we log the fact that PurgeCaches was unable to actually purge?

Reply:
The resource_context_ might not exist (yet), or, on a given platform, might not even be used.

Reply:
This code has been removed now anyway, as the io manager doesn't use caching.